hypervisor/arch/x86/Kconfig

```diff
@@ -40,6 +40,25 @@ config PARTITION_MODE
 
 endchoice
 
+choice
+	prompt "I/O requests completion mode"
+	default IOREQ_NOTIFICATION
+	help
+	  Selete the mode of I/O requests completion
+
+config IOREQ_NOTIFICATION
+	bool "Notification mode"
+	help
+	  SOS will notify HV when I/O request is completed via hypercall.
+
+config IOREQ_POLLING
+	bool "Polling mode"
+	help
+	  SOS only marks I/O request as completed, HV will poll the completion
```

Comment:
poll the completion status

Reply:
Thanks @shiqingg for polishing the description. Updated.